### ISSUE TITLE
[Question] Why phase 1 and 2 in BERT Pre-training

### ISSUE THREAD

**Issue by FrankLeeeee** [NVIDIA/DeepLearningExamples#1127]
In the BERT paper, the model is trained on sequence 512. However, in the [example](https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/LanguageModeling/BERT), there are two phases where the model is trained on sequence length 128 in phase 1 and 512 in phase 2. May I know how much impact there is if only trained with phase 2 directly.

**Comment by gugarosa** [comment]
It is a trick to avoid computational burden of training very large sequence lengths from scratch: https://github.com/google-research/bert#pre-training-tips-and-caveats

**Comment by sharathts** [comment]
Yes, that is correct. Has no accuracy impact, reducing computation significantly. Training phase1 is ~2.7x faster than training phase2.
